shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -229,6 +232,18 @@ fn create_query(messages: &Messages, query: &str, version: Version) -> Result<Me
 
 impl CassandraSinkCluster {
     async fn send_message(&mut self, mut messages: Messages) -> ChainResponse {
+        if let Some(message) = messages.first() {
```

Comment:
Does this happen for every message? I think we only need to set this once. Once a Cassandra connection starts up the version won't change.

Reply:
Perhaps the version field should be an option and we check if it's none before setting it.

Reply:
yeah, thats reasonable, I'll change the field to an Option.